### PR TITLE
Task-41617 : Enlarge the user mangement table by adding new container

### PR DESCRIPTION
**Pull request by ahmadmido**
This fix will enlarge the user management table by adding a container.

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less [hunk 1]
```diff
@@ -99,7 +99,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     }
   }
 }
-#singlePageApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
+#singlePageApplicationContainer, #singlePageUserManegementApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
```
Comment (plamarque) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less]:
```suggestion
#singlePageApplicationContainer, #singlePageUserManagementApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
```

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less [hunk 2]
```diff
@@ -109,6 +109,18 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   }
 }
 
+@media (min-width: @minLargeDesktop) {
+  #singlePageUserManegementApplicationContainer {
```
Comment (plamarque) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less]:
```suggestion
  #singlePageUserManagementApplicationContainer {
```

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less [hunk 3]
```diff
@@ -109,6 +109,18 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+    padding: 20px 5%;
+  }
+}
+
+@media (max-width: @maxDesktop) {
+  #singlePageUserManegementApplicationContainer {
```
Comment (plamarque) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less]:
```suggestion
  #singlePageUserManagementApplicationContainer {
```

**Review by plamarque — DISMISSED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less [hunk 4]
```diff
@@ -99,7 +99,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     }
   }
 }
-#singlePageApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
+#singlePageApplicationContainer, #singlePageUserManagementApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
```
Comment (rdenarie) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less]: does this modification is still needed ?

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less [hunk 5]
```diff
@@ -109,6 +109,18 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+    padding: 20px 1%;
+  }
+}
+
+@media (max-width: @maxDesktop) {
+  #singlePageUserManagementApplicationContainer {
+    margin: 22px 1% 10px;
+  }
+}
+
```
Comment (rdenarie) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less]: does this modification is still needed ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**